docs/ami_lod_rec

```diff
@@ -0,0 +1,203 @@
+
+For every JSON key/element in your metadata that you need to process the LoD Reconciled data into, you need to specify in your Template that data for this element will be read from the 'Processed Data' LoD information. 
+
+In the following example Twig snippet, the "subject_loc" JSON key will map corresponding values from the 'Processed Data' (data.lod) LoD information into a newly created Digital Object/Collection during the AMI Set Processing.
+
+```shell
+"subject_loc": {{ data_lod.mods_subject_topic.loc_subjects_thing|json_encode|raw }},
+```
+        - "subject_loc" = destination JSON Key or Property for your LoD values
+        - data.lod = directs the Twig template to source from the Processed Data LoD information (instead of the original AMI Source CSV accessed via 'data.xx_property_name')
+		- mods.subject.topic = the Column header in the original AMI Source CSV
+		- loc_subjects_thing = the Column containing the Label and URI/L pairs in the Processed Data LoD editable Table/Form (and reference CSV)
+
+The same general pattern can be adapted to apply to different mapping scenarios (original CSV source columns to Reconciled LoD Sources) as needed.
+
+??? info "Full list of Column Options => Corresponding LoD Sources"
+
+- 'loc_subjects_thing' => LoC subjects(LCSH)
+- 'loc_names_thing' => LoC Name Authority File (LCNAF)
+- 'loc_genreForms_thing' => LoC Genre/Form Terms (LCGFT)
+- 'loc_graphicMaterials_thing' => LoC Thesaurus of Graphic Materials (TGN)
+- 'loc_geographicAreas_thing' => LoC MARC List for Geographic Areas
+- 'loc_relators_thing' => LoC Relators Vocabulary (Roles)
+- 'loc_rdftype_CorporateName' => LoC MADS RDF by type: Corporate Name
+- 'loc_rdftype_PersonalName' => LoC MADS RDF by type: Personal Name
+- 'loc_rdftype_FamilyName' => LoC MADS RDF by type: Family Name
+- 'loc_rdftype_Topic' => LoC MADS RDF by type: Topic
+- 'loc_rdftype_GenreForm' =>  LoC MADS RDF by type: Genre Form
+- 'loc_rdftype_Geographic' => LoC MADS RDF by type: Geographic
+- 'loc_rdftype_Temporal' =>  LoC MADS RDF by type: Temporal
+- 'loc_rdftype_ExtraterrestrialArea' => LoC MADS RDF by type: Extraterrestrial Area :space_invader:
+- 'viaf_subjects_thing' => VIAF
+- 'getty_aat_fuzzy' => Getty aat Fuzzy
+- 'getty_aat_terms' => Getty aat Terms
+- 'getty_aat_exact' => Getty aat Exact Label Match
+- 'wikidata_subjects_thing' => Wikidata Q Items
+
+## Step 7: AMI Set Processing Preparation
+
+When you have finished preparing your Template(s) as needed and completed your LoD Reconiliation Review, your AMI Set with LoD Reconciled Data will now need to be Processed.
+
+To Process this set, navigate to the `Process` tab. You will have multiple options related to the Processing outcome for your AMI Set. 
+- Select the "Desired ADOS Statuses After Process" for both Digital Objects (and Digital Object Collections, if also part of your ingest).
```

Comment:
Very thorough @alliomeria!

Do you think it would be helpful to add in that users may choose multiple options at a time when at the select "Desired ADOS Statuses After Process" step?